src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class Ball {
+    private Color color;
+    private int number;
+
+    Ball(Color color, int number) {
```

Comment:
better specify explicitly access modifiers:
```suggestion
    public Ball(Color color, int number) {
```